### DISCUSSION TITLE
chatlog reply information issue

### DISCUSSION THREAD

**Discussion by Tqeslib** [Sehelitar/Kick.bot#71] [Q&A]
Hello, it's me again. I use write to file to keep ChatLog. It keeps accurate messages and their sending dates. However, if a message is a reply, it does not show to whom it is replied. The fights that occur in almost every chat also occur in my friend's chat. That's why reply information is very important. How can I get reply information? Here is my write to file method.
![Screenshot_20](https://github.com/user-attachments/assets/29de3f39-277d-48ac-8c3e-7bfee2f3bb31)

**Comment by Sehelitar** [comment]
Hi! 👋 
Some new arguments will be available in the next release : https://github.com/Sehelitar/Kick.bot/wiki/Triggers#chat-message
![image](https://github.com/user-attachments/assets/a553b5a8-6052-4e69-a3dd-59eb34fc053a)

This is already available in beta with Streamer.bot 0.2.4-beta and Kick.bot 0.5+ -- **BUT** -- I strongly recommend that you wait for the public release to come out, unless you are already using it.

**Comment by Tqeslib** [comment]
Thank you for your efforts. Should I add these arguments directly? Or should I open another write to file sub-action?
